cmd/vic-machine/create/create.go

```diff
@@ -239,6 +241,12 @@ func (c *Create) Flags() []cli.Flag {
 			Usage:       "Virtual Container Host x509 certificate file",
 			Destination: &c.cert,
 		},
+		cli.StringFlag{
+			Name:        "base-image-size",
```

Comment:
The option name is just a little confusing to me. Do you guys have any other suggestions?